src/components/Image/Image.js

```diff
@@ -5,7 +5,7 @@ const Image = ({ title, image }) => {
   return (
     <article className="image-container">
       <h2 className="movie-title">{title}</h2>
-      <img className="backdrop-image" src={image} />
+      <img className="backdrop-image" src={image} alt="movie poster" />
```

Comment:
hooray accessibility! Might be worth using interpolation to pull in the movie's title into the alt tag, as well!